### PR TITLE
OBSDOCS-481 - Logging attributes update

### PR DESCRIPTION
**Pull request by libander**
Version(s): 4.11+

Issue: [OBSDOCS-481](https://issues.redhat.com//browse/OBSDOCS-481)

Link to docs preview: https://70801--ocpdocs-pr.netlify.app/openshift-enterprise/latest/logging/cluster-logging

QE review: @anpingli / @kabirbhartiRH / @QiaolingTang 
- [x] QE has approved this change.

Additional information: Attribute changes only, QE approval should not be necessary.

### CODE REVIEW

File: logging/cluster-logging-support.adoc [hunk 1]
```diff
@@ -9,9 +9,9 @@ toc::[]
 include::snippets/logging-supported-config-snip.adoc[]
 include::snippets/logging-compatibility-snip.adoc[]
 
-The {logging-title} is an opinionated collector and normalizer of application, infrastructure, and audit logs. It is intended to be used for forwarding logs to various supported systems.
+{logging-uc} {for} is an opinionated collector and normalizer of application, infrastructure, and audit logs. It is intended to be used for forwarding logs to various supported systems.
```
Comment (mburke5678) [inline logging/cluster-logging-support.adoc:12]: The variables in this file are not rendering in preview.

File: logging/cluster-logging-uninstall.adoc [hunk 2]
```diff
@@ -6,7 +6,7 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can remove the {logging} from your {product-title} cluster by removing installed Operators and related custom resources (CRs).
+You can remove {logging} from your {product-title} cluster by removing installed Operators and related custom resources (CRs).
```
Comment (mburke5678) [inline logging/cluster-logging-uninstall.adoc:9]: The variables in this file are not rendering in preview.

File: logging/cluster-logging.adoc [hunk 3]
```diff
@@ -11,9 +11,9 @@ As a cluster administrator, you can deploy {logging} on an {product-title} clust
 
 include::snippets/logging-kibana-dep-snip.adoc[]
 
-{product-title} cluster administrators can deploy the {logging} by using Operators. For information, see xref:../logging/cluster-logging-deploying.adoc#cluster-logging-deploying[Installing the {logging-title}].
+{product-title} cluster administrators can deploy the {logging} by using Operators. For information, see xref:../logging/cluster-logging-deploying.adoc#cluster-logging-deploying[Installing {logging}].
```
Comment (mburke5678) [inline logging/cluster-logging.adoc:14]:
```suggestion
{product-title} cluster administrators can deploy {logging} by using Operators. For information, see xref:../logging/cluster-logging-deploying.adoc#cluster-logging-deploying[Installing {logging}].
```

File: logging/config/cluster-logging-configuring.adoc [hunk 4]
```diff
@@ -9,7 +9,7 @@ toc::[]
 {logging-title-uc} is configurable using a `ClusterLogging` custom resource (CR) deployed
 in the `openshift-logging` project.
 
-The {logging} operator watches for changes to `ClusterLogging` CR,
+The {clo] watches for changes to `ClusterLogging` CR,
```
Comment (mburke5678) [inline logging/config/cluster-logging-configuring.adoc:12]: Is this file being used?

File: logging/dedicated-cluster-logging.adoc [hunk 5]
```diff
@@ -1,7 +1,7 @@
 :_mod-docs-content-type: ASSEMBLY
 :context: dedicated-cluster-logging
 [id="dedicated-cluster-logging"]
-= Configuring the {logging-title}
+= Configuring {logging}
```
Comment (mburke5678) [inline logging/dedicated-cluster-logging.adoc:4]: Is this file being used?

File: logging/logging_alerts/default-logging-alerts.adoc [hunk 6]
```diff
@@ -6,7 +6,7 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-Logging alerts are installed as part of the {clo} installation. Alerts depend on metrics exported by the log collection and log storage backends. These metrics are enabled if you selected the option to *Enable operator recommended cluster monitoring on this namespace* when installing the {clo}. For more information about installing logging Operators, see xref:../../logging/cluster-logging-deploying#cluster-logging-deploy-console_cluster-logging-deploying[Installing the {logging-title} using the web console].
+Logging alerts are installed as part of the {clo} installation. Alerts depend on metrics exported by the log collection and log storage backends. These metrics are enabled if you selected the option to *Enable operator recommended cluster monitoring on this namespace* when installing the {clo}. For more information about installing logging Operators, see xref:../../logging/cluster-logging-deploying#cluster-logging-deploy-console_cluster-logging-deploying[Installing {logging} using the web console].
```
Comment (mburke5678) [inline logging/logging_alerts/default-logging-alerts.adoc:9]: The variables in this file are not rendering in preview.

File: modules/cluster-logging-configuring-image-about.adoc [hunk 7]
```diff
@@ -5,7 +5,7 @@
 [id="cluster-logging-configuring-image-about_{context}"]
 = Understanding {logging} component images
 
-There are several components in the {logging-title}, each one implemented with one or more images. Each image is specified by an environment variable
+There are several components in {logging}, each one implemented with one or more images. Each image is specified by an environment variable
```
Comment (mburke5678) [inline modules/cluster-logging-configuring-image-about.adoc:8]: This file is not being used. Commented out in cluster-logging-configuring.adoc

File: modules/cluster-logging-deploying-about.adoc [hunk 8]
```diff
@@ -5,7 +5,7 @@
 
 :_mod-docs-content-type: CONCEPT
 [id="cluster-logging-deploying-about_{context}"]
-= About deploying and configuring the {logging-title}
+= About deploying and configuring {logging}
```
Comment (mburke5678) [inline modules/cluster-logging-deploying-about.adoc:8]: Is this file being used?

File: modules/gitops-storing-and-retrieving-argo-cd-logs.adoc [hunk 9]
```diff
@@ -11,7 +11,7 @@ You can use the Kibana dashboard to store and retrieve Argo CD logs.
 .Prerequisites
 
 * The {gitops-title} Operator is installed in your cluster.
-* The {logging-title} is installed with default configuration in your cluster.
+* {logging-uc} is installed with default configuration in your cluster.
```
Comment (mburke5678) [inline modules/gitops-storing-and-retrieving-argo-cd-logs.adoc:14]: Is this file being used?

File: snippets/logging-log-types-snip.adoc [hunk 10]
```diff
@@ -6,7 +6,7 @@
 //
 :_mod-docs-content-type: SNIPPET
 
-The {logging-title} collects container logs and node logs. These are categorized into types:
+{logging-uc} collects container logs and node logs. These are categorized into types:
```
Comment (mburke5678) [inline snippets/logging-log-types-snip.adoc:9]: Is this file being used?

File: snippets/logging-loki-vs-lokistack-snip.adoc [hunk 11]
```diff
@@ -6,4 +6,4 @@
 //
 :_mod-docs-content-type: SNIPPET
 
-In logging subsystem documentation, LokiStack refers to the logging subsystem supported combination of Loki, and web proxy with OpenShift Container Platform authentication integration. LokiStack’s proxy uses OpenShift Container Platform authentication to enforce multi-tenancy. Loki refers to the log store as either the individual component or an external store.
+In logging documentation, LokiStack refers to the supported combination of Loki and web proxy with OpenShift Container Platform authentication integration. LokiStack’s proxy uses OpenShift Container Platform authentication to enforce multi-tenancy. Loki refers to the log store as either the individual component or an external store.
```
Comment (mburke5678) [inline snippets/logging-loki-vs-lokistack-snip.adoc:9]: Is this file being used?

**Comment by mburke5678** [comment]
@libander I found a number of places in the previews where the variables were not rendered. 
I made note of those. It would be helpful if you gave hard links to each file for review. I know there are a lot of them; but it would making checking these things easier.

**Comment by libander** [comment]
> @libander I found a number of places in the previews where the variables were not rendered. I made note of those. It would be helpful if you gave hard links to each file for review. I know there are a lot of them; but it would making checking these things easier.

Do they normally render? When they're new variables as far as I can recall I haven't seen a local build or preview that successfully rendered them, since the change hasn't merged yet.

**Comment by libander** [comment]
> It would be helpful if you gave hard links to each file for review. I know there are a lot of them; but it would making checking these things easier.

I hear you, but this was a change to the entirety of the logging docs, and not all files are necessarily even going to show up in a preview as some are snippets not currently in use. If its any help, I did document my steps to ensure the changes were thorough: 

Searched the repo for instances of:
{logging-title}
The {logging-title}
the {logging-title]
logging(?=.*installed) [regex]

File: logging/cluster-logging-support.adoc (hunk 1 above)
Comment (libander) [reply logging/cluster-logging-support.adoc:12]: Fixed!

File: logging/cluster-logging-uninstall.adoc (hunk 2 above)
Comment (libander) [reply logging/cluster-logging-uninstall.adoc:9]: Fixed!

File: logging/logging_alerts/default-logging-alerts.adoc (hunk 6 above)
Comment (libander) [reply logging/logging_alerts/default-logging-alerts.adoc:9]: Fixed!

File: modules/cluster-logging-clo-status-comp.adoc [hunk 12]
```diff
@@ -21,7 +21,7 @@ You can view the status for a number of {logging} components.
 $ oc project openshift-logging
 ----
 
-. View the status of the {logging-title} environment:
+. View the status of {logging} environment:
```
[the remark this answers is not in the document]
Comment (libander) [reply modules/cluster-logging-clo-status-comp.adoc:24]: Fixed!

**Comment by bscott-rh** [comment]
Merge review LGTM.

**Pull request merged**